website/docs/r/lb_rule.html.markdown

```diff
@@ -67,6 +66,7 @@ The following arguments are supported:
 * `load_distribution` - (Optional) Specifies the load balancing distribution type to be used by the Load Balancer. Possible values are: `Default` – The load balancer is configured to use a 5 tuple hash to map traffic to available servers. `SourceIP` – The load balancer is configured to use a 2 tuple hash to map traffic to available servers. `SourceIPProtocol` – The load balancer is configured to use a 3 tuple hash to map traffic to available servers. Also known as Session Persistence, where  the options are called `None`, `Client IP` and `Client IP and Protocol` respectively.
 * `disable_outbound_snat` - (Optional) Is snat enabled for this Load Balancer Rule? Default `false`.
 * `enable_tcp_reset` - (Optional) Is TCP Reset enabled for this Load Balancer Rule? Defaults to `false`.
+* `resource_group_name` - (Optional / **Deprecated**) The name of the resource group in which to create the resource.
```

Comment:
which means this is no longer forcenew

Reply:
Here is using [SchemaResourceGroupNameDeprecated() ](https://github.com/terraform-providers/terraform-provider-azurerm/blob/326ae0569f803b4fbaee7d3a40aaa56ee6568ce8/azurerm/helpers/azure/resource_group.go#L21)which doesn't include forcenew.